CONTRIBUTING.md

```diff
@@ -48,6 +48,20 @@ We have a pull request template setup that you will fill out when you open your
 
 > We try to respond to all pull requests and issues within 7 days. We welcome feedback from everyone involved in the project in open pull requests. 
 
+#### Building binaries
```

Comment:
Not sure this is relevant to how to contribute.

Reply:
We need actual documentation, but it was this or shove it in the already crowded readme. If folks want to build the project to see how it works before contributing then this is legit information for them. When we have actual docs we can put it there.